### PR TITLE
Feat/automated releases

### PR DESCRIPTION
**Pull request by SorsOps**
- Adds support for improving the release flow through github workflows
- Adds additional sentry tracking 
- Setups up changesets in anticipation of monorepo support for the tokens-transformer
- Removes old unnecessary shell scripts for Sentry and includes them into the webpack build process

### CODE REVIEW

File: src/app/sentry.ts [hunk 1]
```diff
@@ -0,0 +1,51 @@
...
+  switch (process.env.ENVIRONMENT) {
+    case 'production':
+    case 'beta':
+      Sentry.addTracingExtensions();
+      Sentry.init({
+        dsn: DSN,
+        release: `figma-tokens@${pjs.version}`,
+        environment: process.env.ENVIRONMENT,
+        tracesSampleRate: SAMPLING,
+        // @ts-ignore This is correct, but the types are wrong.
```
Comment (roppazvan) [inline src/app/sentry.ts:39]: I've recently read about ts-expect-error. Wouldn't this be more suitable for these cases?

File: src/app/store/models/settings.tsx [hunk 2]
```diff
@@ -30,6 +31,10 @@ export interface SettingsState {
   shouldSwapStyles: boolean;
   baseFontSize: string;
   aliasBaseFontSize: string;
+  /**
+   * Whether the user has opted in for session recording in Sentry
+   */
+  sessionRecording:boolean;
```
Comment (roppazvan) [inline src/app/store/models/settings.tsx:37]: do you have an issue with prettier ? it's weird that theres no space here

**Review by roppazvan — APPROVED** [review]
LGTM

File: src/app/store/models/settings.tsx (hunk 2 above)
Comment (SorsOps) [reply src/app/store/models/settings.tsx:37]: Yeah that is strange, I don't get any indication about it in any of my prettier linting either. Regardless I will amend

File: src/app/sentry.ts (hunk 1 above)
Comment (roppazvan) [reply src/app/sentry.ts]: yes, and when using ts-expect-error ts will let us know when the types are fixed, and we will have to remove the ts-expect-error

**Pull request merged**